src/main/webapp/app/exercises/shared/exercise-headers/header-exercise-page-with-details.component.scss

```diff
@@ -0,0 +1,5 @@
+@import 'src/main/webapp/content/scss/artemis-variables';
+
+.highlighted-deadline {
+    font-size: 1.575rem; // same as h3
```

Comment:
If it's the same as h3, you could maybe even just use the Bootstrap class `h3`, right?

Reply:
I didn't know that this can also be used as a class. Thanks for the info!